Add conferences to sponsor section in footer

klappradla opened the pull request:
I added an additional _"conferences"_ section to the footer, listing the first batch of confirmed conferences who offer free tickets for students. It is includes as a partial within the _"sponsors"_ section of the footer and therefore displayed on all section where the parent section is show (`sponsors: true` is the option as far as I remember).

I used the existing `conferences.yml` file for the data (so conferences are not listed as part of the sponsor's page) and the existing`img/confs` directory for the images. I'm not a Photoshop-hero and did not see any guidelines on how the images should look like, so I just tried to adapt the style and size of the other images in the footer.

Not 100% sure if that's what you expected it to look and work @mkalininait. Just tell me whatever I should change and I'll update this PR.

klappradla committed:
Add conferences to sponsor section in footer

This closes #267 by adding a separate conferences-section to the
footer.

mkalininait commented:
Hi @klappradla Thanks for your work! 
I found logos of better quality (those ones which I linked before were 200x200, my bad).
I adjusted sizes. Here they are.
![codemowarsaw-logo](https://cloud.githubusercontent.com/assets/1745735/17743306/8da3ab1a-64a3-11e6-8da8-3c3594d61ce1.png)
![connect-tech-logo](https://cloud.githubusercontent.com/assets/1745735/17743303/8d9d34ba-64a3-11e6-90e6-82e8c1aa0b61.png)
![euruko-logo](https://cloud.githubusercontent.com/assets/1745735/17743307/8da8f6d8-64a3-11e6-94b3-053ac4d2fff0.png)
![goto-cph-logo](https://cloud.githubusercontent.com/assets/1745735/17743304/8d9db520-64a3-11e6-8d52-4ea223675c32.png)
![logo-ruby-bg 2x](https://cloud.githubusercontent.com/assets/1745735/17743308/8da95538-64a3-11e6-9f31-5dffd0ccb33a.png)
![logo-site-branco](https://cloud.githubusercontent.com/assets/1745735/17743305/8da17020-64a3-11e6-84bf-06e5577df1b6.png)
![rubyconf-au-log-2](https://cloud.githubusercontent.com/assets/1745735/17743309/8dc0f0c6-64a3-11e6-8fba-365e7d94807b.png)
![strange-loop-logo](https://cloud.githubusercontent.com/assets/1745735/17743310/8dc37328-64a3-11e6-94ea-600e237c45eb.jpg)

klappradla committed:
update conference logos

This updates the images used in the conferences section in the footer to
be more up 2 date and of better quality.
